content/tutorials/wallets.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Wallet Basics
+description: 
+---
+
+Ocean users need a wallet to manage their ETH and OCEAN tokens. 
+
+## Recommendations
+
+- **Easiest:** Use MetaMask. See the tutorial about [how to set up MetaMask](/tutorials/metamask-setup/). OR
```

Comment:
that tutorial link does not exist anymore

Reply:
fixed